### PR TITLE
DM-43122: Added jmin to zernike utils.

### PR DESCRIPTION
**Pull request by jfcrenshaw**
You can now set a custom `jmin` on any of the Zernike utils. This is useful for `ts_imsim`.

### CODE REVIEW

**Comment by teweitsai** [comment]
If the `jmin` is 0 or -1, the code will throw out an error or not?

File: python/lsst/ts/wep/utils/zernikeUtils.py [hunk 1]
```diff
@@ -407,29 +428,32 @@ def convertZernikesToPsfWidth(
...
     axis : str, optional
         The axis for which to return the parity. Can be "x" or "y".
+        (the default is "x")
 
     Returns
     -------
     np.ndarray
-        The numpy array of parities, starting with Noll index 4.
-        +1 corresponds to even parity, while -1 corresponds to odd parity.
+        Aarray of parities, with +1 corresponding to even parity,
```
Comment (jbkalmbach) [inline python/lsst/ts/wep/utils/zernikeUtils.py]: typo `Aarray`

**Review by jbkalmbach — APPROVED** [review]
Looks good, one docstring typo. See Te-Wei's comment about errors.

File: python/lsst/ts/wep/utils/zernikeUtils.py (hunk 1 above)
Comment (jfcrenshaw) [reply python/lsst/ts/wep/utils/zernikeUtils.py]: Nice catch! Fixed.

**Comment by jfcrenshaw** [comment]
@teweitsai jmin must be >= 0 and jmax must be >= jmin. I added a note on this to all of the docstrings, plus checks on these values. Thanks!

**Pull request merged**